guacamole-docker/bin/start.sh

```diff
@@ -707,6 +722,11 @@ END
     exit 1;
 fi
 
+# Use TOTP if specified.
+if [ -n "$TOTP_ENABLED" ]; then
```

Comment:
I'm not a huge fan of having to add an environment variable that doesn't map to a property, as it deviates from established patterns, but since the TOTP auth doesn't have any required properties I'm not sure we have any choice.

Reply:
FWIW, I'm not tied to this - if we want to change it to something else or some other way of enabling it, I'm open to the options.  It just seemed a little weird to me to trigger it by adding a parameter that isn't required.  But, I'm willing to go with whatever.

Reply:
The only "generic" solution I see is :
`if env | grep -q "^TOTP_"; then`
So that enforcing at least one of the `TOTP` properties would enable it.

Reply:
@mike-jumper Thoughts, here?  I'm not tied to my suggestion, so if you have a vote...?

Reply:
My preference would be:

* Continue using `TOTP_ENABLED`.
* Actually check the value rather than simply whether it's set. The current implementation would enable TOTP if the user sets `TOTP_ENABLED` to "1", to "false", to "salmon", etc., and that's not exactly rigorous.

I'm not against additionally accepting `TOTP_*` in lieu of `TOTP_ENABLED` for convenience, but if so:

* Only actual, known variables should be accepted (for similar reasons to the above... setting `TOTP_FISH` to "salmon" shouldn't result in TOTP being enabled).
* `TOTP_ENABLED` should still probably be accepted (and checked), as users shouldn't be required to set a TOTP-related variable to its default value just to enable use of TOTP.

Reply:
Thank you for your feedbacks.
I then updated to :
`if [ "$TOTP_ENABLED" = "true" ]`